internal/cli/user_blocks.go

```diff
@@ -0,0 +1,110 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+	"gopkg.in/auth0.v5/management"
+)
+
+var (
+	userID = Argument{
+		Name: "user_id",
+		Help: "user_id of the user.",
+	}
+)
+
+func userBlocksCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "user-blocks",
+		Short: "Manage brute-force protection user blocks.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listUserBlocksByUserIdCmd(cli))
+	cmd.AddCommand(deleteUserBlocksByUserIdCmd(cli))
+	return cmd
+}
+
+func listUserBlocksByUserIdCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		user_id string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "listByUserId",
```

Comment:
Please use `list`. Entire command should be `auth0 users blocks list`.

Reply:
we can list by `user_id` or list by `email / phonenumber / user_identifier`. It's a different API call, what would a good way to name the handle the two different input types?

Reply:
The command should accept any of these as a single argument, and then disambiguate. It's the easiest for the user, that remains unaware (as they should) of those implementation details.

Reply:
e.g. `auth0 apis show` accepts an API id or the audience.

Reply:
For the end user, there's no difference.